src/applications/vre/28-8832/containers/ConfirmationPage.jsx

```diff
@@ -23,7 +23,10 @@ export class ConfirmationPage extends React.Component {
   render() {
     const { submission, data } = this.props.form;
     const { response } = submission;
-    const name = data.veteranFullName;
+
+    const name = data.veteranInformation.fullName
+      ? data.veteranInformation.fullName
+      : data.fullName;
```

Comment:
Is this the right check? If a user is logged out, and filling out the form as a dependent, then this would show the veteran's name instead of the claimant's name..right? I think we always want to just show whatever is in `data.fullName`. 

If a user is logged in, then `fullName` is undefined as it's never filled out, so no name would be displayed. I think we need to check `isLoggedIn` in the user object via redux, and fill the name from the user object if a user is indeed authenticated.

Reply:
Moved this work into this ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/14856